api/envoy/config/cluster/redis/redis_cluster.proto

```diff
@@ -0,0 +1,50 @@
+
+
+syntax = "proto3";
+
+package envoy.config.cluster.redis;
+
+option java_outer_classname = "RedisClusterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.cluster.redis";
+option go_package = "v2";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Redis Cluster Configuration]
+// To properly support Redis Cluster it is necessary to store the current topology of the cluster.
+// A random node in the cluster can be queried for the topology using the
```

Comment:
I'm a little unclear on what this is saying. Is this what the cluster does? Can you clarify a bit for a reader who might not understand the background of Redis Cluster?

Reply:
I will add some explanation of what Redis Cluster does, and links, and how this compares  to Twemproxy.